### PR TITLE
Added DP Device.

### LINKED ISSUE #230: DPDEV
Not really sure what this device does. It has something to do with 7-bit conversions, I think.

DEVICE; JOBDEV DP
AR5:ALAN; DPDEV 19

### PR DESCRIPTION
**Pull request by eswenson1**
Resolves #230

### CODE REVIEW

**Comment by larsbrinkhoff** [comment]
Error:

    *:midas device;jobdev dp_ar5:alan;dpdev 19
    AR5:DPDEV 19 ALAN;
    NO SUCH DEVICE

The ALAN directory isn't added to the sources tape.

**Comment by larsbrinkhoff** [comment]
Fixed.

**Comment by eswenson1** [comment]
Thanks for fixing that.

File: build/build.tcl [hunk 1]
```diff
@@ -509,6 +509,9 @@ respond "*" ":link sys2;ts chtn,sysbin;supdup bin\r"
 respond "*" ":midas sys;ts ttloc_sysen1;ttloc\r"
 expect ":KILL"
 
+respond "*" ":midas device;jobdev dp_ar5:alan;dpdev 19\r"
```
Comment (larsbrinkhoff) [inline build/build.tcl]: Did you have to specify the version number?

**Comment by eswenson1** [comment]
No. That was a mistake.

**Comment by larsbrinkhoff** [comment]
I fixed it in the lars/xfile branch.

**Pull request merged**